bin/common.ml

```diff
@@ -134,6 +134,22 @@ let term =
          & flag
          & info ["debug-backtraces"] ~docs
              ~doc:{|Always print exception backtraces.|})
+  and+ term_persistence =
+    let arg = Arg.(conv
```

Comment:
We should use `Arg.enum` here (as done for the `--display` option)

Reply:
Done